index.js

```diff
@@ -56,17 +56,19 @@ module.exports = function (namespace) {
 			return subLogger[n];
 		},
 		info: log.bind(log, 'info', 'log'),
-		json: function () {
+		json: function() {
 			for (let i = 0; i < arguments.length; i++) {
 				log("info", "log", JSON.stringify(arguments[i], null, 2));
 			}
 		},
 		time: log.bind(log, 'time', 'time'),
 		timeEnd: log.bind(log, 'time', 'timeEnd'),
 		log: log.bind(log, 'info', 'log'),
-		debug: log.bind(log, 'debug', 'log'),
+		debug: log.bind(log, 'debug', 'debug'),
```

Comment:
@jgrantr do you see this causing any issues. Switching debug to bind to console.debug

Reply:
@czirker Just that we are aliasing an already aliased function. I think we should leave it as-is (bind to console.log)